Automattic/wp-calypso
# Properly indent nested lists in CONTRIBUTING.md

#### Changes proposed in this Pull Request

* There are parts of CONTRIBUTING.md that read like they are meant to be detailed explanations of the steps they come after. In my opinion it reads better (and makes it easier to scan) if they are indented properly such that they become nested lists, as opposed to remaining as bullets at the top level.

#### Testing instructions



* Change is in a text file, and purely cosmetic. The only thing to verify is whether you agree with it.

Before:
<img width="657" alt="ContribBefore" src="https://user-images.githubusercontent.com/3772093/56451535-115d4680-62e1-11e9-8c9b-fea01b1ccf21.png">

After:
<img width="607" alt="ContribAfter" src="https://user-images.githubusercontent.com/3772093/56451541-15896400-62e1-11e9-8255-12fdc927bea5.png">

cc @drw158, @codebykat: I believe this is ready for review
